javascript/ql/src/Security/CWE-078/UselessUseOfCat.ql

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Useless use of cat
+ * @description Using cat to simply read a file can lead to unintended bugs, and at worst security issues.
```

Comment:
Can we turn this up a notch? We should also have a sentence for each of these bad properties in the qhelp.


```suggestion
 * @description Using `cat`-process to simply read a file is unnecessarily complex, inefficient, unportable, can lead to subtle bugs, or even security vulnerabilities.
```